inc/db-magdb-open.inc.php

```diff
@@ -7,6 +7,6 @@
 
 #Connect to DB
 $PGLINK = pg_connect("host=$PG_HOST user=$PG_USER password=$PG_PASS dbname=$PG_NAME")
-or trigger_error("Unable to open connection to magDB", E_USER_ERROR);
+or error("Unable to conect to:", $PG_HOST);
```

Comment:
Seeing as this isn't an error log dump, I'll make it show the name of the service instead of the full hostname.